esx_service/utils/vmdk_utils.py

```diff
@@ -307,6 +311,32 @@ def get_vm_config_path(vm_name):
     vm_config_path = os.path.join(datastore_path, path)
     return vm_config_path
 
+def check_volumes_mounted(vm_list):
+    """
+    Return error_info any vm in @param vm_list have docker volume mounted
+    """
+    for vm_id in vm_list:
+        vm = vmdk_ops.findVmByUuid(vm_id)
+        if vm:
+            for d in vm.config.hardware.device:
+                # if device is not a virtual disk, skip this device
+                if type(d) != vim.vm.device.VirtualDisk:
+                    continue
+
+                # Filename format is as follows:
+                # "[<datastore name>] <parent-directory>/tenant/<vmdk-descriptor-name>"
+                # Trim the datastore name and keep disk path.
+                _, disk_path = d.backing.fileName.rsplit("]", 1)
+                logging.info("backing disk name is %s", disk_path)
+                # If disk path start with "dockvols", the disk is a docker volume
```

Comment:
Can we extract the logic of checking whether a disk is a docker volume (322-332) to a separate method?